test/base_decorator_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../capitalize_decorator'
+require_relative '../person'
+require_relative '../trimmer_decorator'
+
```

Comment:
- [ ] It would be nice if you follow the project requirements which says there should be at least one file for each class and I saw you are missing some files such as `student_spec.rb`, `teacher_spec.rb`, `rental_spec.rb` 😨 Kindly add these files and tests for these classes 👍 to follow the project requirements 😊

Reply:
Since I have added a test for `person.rb` which carries both student and teacher data that is why I didn't add separate test, but I will sure add